prisma/seed.ts

```diff
@@ -15,13 +15,20 @@ async function main() {
   //   },
   // });
   // console.log({ user });
-
-  const lesson1 = await prisma.lessons.create({
+  // const lesson1 = await prisma.lessons.create({
+  //   data: {
+  //     quizFileName: "lesson-4-quiz.json",
+  //   },
+  // });
+  // console.log({ lesson1 });
+  const completed2 = await prisma.completedQuizzes.create({
     data: {
-      quizFileName: "quiz-lesson-1.json",
+      userId: "cll3hcuim00001wujlay766tk",
```

Comment:
not sure what this is; anything sensitive about this ID? should it be passed in dynamically or via env var?

Reply:
> not sure what this is; anything sensitive about this ID? should it be passed in dynamically or via env var?

No no, this db query with prisma was used to populate my DB, and with the boolean test the `completed` state, instead or solving the Quiz :stuck_out_tongue_winking_eye:  :stuck_out_tongue_winking_eye: